Aperture fails to build with OpenAPI31 feature

Not sure if this is from the inline imports change or what happened in the last couple of weeks but aperture now fails to build in a clean chroot with rustup toolchains throwing:
```rust
error[E0277]: the trait bound `std::string::String: From<jaq_interpret::Filter>` is not satisfied
    --> src/engine/executor.rs:1212:17
     |
1211 |             Err(e) => Err(Error::jq_filter_error(
     |                           ---------------------- required by a bound introduced by this call
1212 |                 filter,
     |                 ^^^^^^ the trait `From<jaq_interpret::Filter>` is not implemented for `std::string::String`
     |
     = help: the following other types implement trait `From<T>`:
               `std::string::String` implements `From<&mut str>`
               `std::string::String` implements `From<&std::string::String>`
               `std::string::String` implements `From<&str>`
               `std::string::String` implements `From<Box<str>>`
               `std::string::String` implements `From<Cow<'_, str>>`
               `std::string::String` implements `From<char>`
               `std::string::String` implements `From<clap::Id>`
               `std::string::String` implements `From<clap::builder::Str>`
             and 8 others
     = note: required for `jaq_interpret::Filter` to implement `Into<std::string::String>`
note: required by a bound in `error::Error::jq_filter_error`
    --> src/error.rs:909:41
     |
 909 |     pub fn jq_filter_error(filter: impl Into<String>, reason: impl Into<String>) -> Self {
     |                                         ^^^^^^^^^^^^ required by this bound in `Error::jq_filter_error`

error[E0308]: `match` arms have incompatible types
   --> src/spec/parser.rs:224:13
    |
220 |       let oas3_spec = match oas3::from_yaml(preprocessed) {
    |                       ----------------------------------- `match` arms have incompatible types
221 |           Ok(spec) => spec,
    |                       ---- this is found to be of type `Spec`
...
224 | /             oas3::from_json(preprocessed).map_err(|e| {
225 | |                 Error::serialization_error(format!(
226 | |                     "Failed to parse OpenAPI 3.1 spec as YAML or JSON: {e}"
227 | |                 ))
228 | |             })?;
    | |                ^
    | |                |
    | |________________help: consider removing this semicolon
    |                  expected `Spec`, found `()`

Some errors have detailed explanations: E0277, E0308.
For more information about an error, try `rustc --explain E0277`.
error: could not compile `aperture-cli` (lib) due to 2 previous errors
warning: build failed, waiting for other jobs to finish...
```
happens with `jq` feature enabled or disabled, prevents use with OpenProject without `openapi31` feature.

--- a/src/engine/executor.rs
+++ b/src/engine/executor.rs
@@ -1209,7 +1209,7 @@ pub fn apply_jq_filter(response_text: &str, filter: &str) -> Result<String, Erro
                 }
             }
             Err(e) => Err(Error::jq_filter_error(
-                filter,
+                format!("{:?}", filter),
                 format!("Filter execution error: {e}"),
             )),
         }

--- a/src/spec/parser.rs
+++ b/src/spec/parser.rs
@@ -225,7 +225,7 @@ fn parse_with_oas3_direct_with_original(
                 Error::serialization_error(format!(
                     "Failed to parse OpenAPI 3.1 spec as YAML or JSON: {e}"
                 ))
-            })?;
+            })?
         }
     };
 

